app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), data: { turbo: false }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+   <h3>Sign up</h3>
+  <div>
+      <%= f.label :name, class: "label-sign"%>
+      <%= f.text_field :name, autofocus: true, autocomplete: "name", class: "input-sign"%>
+  </div>
+
+  <div>
+    <%= f.label :email, class: "label-sign"%>
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "input-sign" %>
+  </div>
+
+  <div>
+    <%= f.label :password, class: "label-sign" %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %>
```

Comment:
- [ ] As already pointed out above, please ensure that the sign up and login pages are close to the original design and are mobile responsive.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/189241035-23110e1b-4e56-4d2b-81c1-840f96690616.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/189241012-0d3aed5b-0e43-4dc7-b35e-8fc243acfe48.png)